config/runtime.exs

```diff
@@ -153,17 +153,10 @@ config :lambda_ethereum_consensus, BeaconApi.Endpoint,
     layout: false
   ]
 
-if is_binary(keystore) and is_binary(keystore_pass) do
-  {pubkey, privkey} = Keystore.decode_from_files!(keystore, keystore_pass)
+config :lambda_ethereum_consensus, LambdaEthereumConsensus.Validator.Supervisor,
+  keystore_dir: keystore_dir,
+  keystore_pass_dir: keystore_pass_dir
```

Comment:
Parsing should be done here to fail fast. We can add a TODO and open an issue if it's too much work.